aws/resource_aws_s3_bucket.go

```diff
@@ -396,6 +400,20 @@ func resourceAwsS3Bucket() *schema.Resource {
 													Type:     schema.TypeString,
 													Optional: true,
 												},
+												"access_control_translation": {
+													Type:     schema.TypeSet,
+													Optional: true,
+													MinItems: 1,
+													MaxItems: 1,
+													Elem: &schema.Resource{
+														Schema: map[string]*schema.Schema{
+															"owner": {
```

Comment:
I'd suggest renaming this to `owner_id` to match similar places in the code base.
Also can add `ValidateFunc: validateAwsAccountId,`.

Reply:
This field is not associated with account IDs so neither of these changes are relevant.

```
    // The override value for the owner of the replica object.
    //
    // Owner is a required field
    Owner *string `type:"string" required:"true" enum:"OwnerOverride"`
```

There will be validation for the enum via:

```
ValidateFunc: validation.StringInSlice([]string{
	s3.OwnerOverrideDestination,
}, false),
```